roles/satosa/defaults/main.yml

```diff
@@ -13,4 +13,13 @@ satosa_port: "8080"
 satosa_state_encryption_key: "poikjh567"
 satosa_user_id_hash_salt: "59o3gaxfwg0svem1n1g3tk22i9mihcom"
 
+satosa_mdui_display_name: SATOSA Display Name
+satosa_mdui_description: SATOSA Description
+satosa_mdui_information_url: "http://information.url/"
+satosa_mdui_privacy_statement_url: "http://privacy.url/"
+satosa_mdui_keywords: "[{lang: nl, text: [sleutelwoord1, sleutelwoord2]}, {lang: en, text: [keyword1, keyword2]}]"
+satosa_mdui_logo: "http://logo.url/"
+satosa_mdui_logo_width: "100"
+satosa_mdui_logo_height: "100"
```

Comment:
I think display_name, description and keywords should be separately configurable for the frontend and backend

Reply:
I don't. It should be totally clear that you're connecting to the SP or IdP from url, organisation display_name, name, SSO descriptor and endpoints... It won't help people who don't understand metadata anyway.